regression/iterate_data.py

```diff
@@ -35,6 +38,8 @@ def parse_args() -> tuple[Namespace, dict[str, str]]:
         tuple(Namespace, dict[str, str]): Command-line arguments and named arguments.
     """
     args = ArgumentParser()
+    args.add_argument('--cloud', type=str)
+    args.add_argument('--check_download', default=False, action='store_true')
     args.add_argument('--local', default=False, action='store_true')
```

Comment:
can we have `local` as `str` instead of `bool` and let the user provides the path of local, if it is not provided then streaming dataset would create a tmp directory